Client/src/Remote/Endpoints/UserEndpoint.tsx

```diff
@@ -0,0 +1,22 @@
+import { IUserAccountModel, IUserDetailsModel, toLocalUserAccountModel } from "../../Models/Authentication/IUserAccountModel"
+import { get, post } from "../RemoteHelper"
+
+import { stringify } from 'query-string'
+
+const resetPasswordRoute = '/resetPassword'
```

Comment:
This one should probably be moved to login

Reply:
+

Reply:
+1

Reply:
done